app/views/components/AnswerSection.scala.html

```diff
@@ -65,20 +62,32 @@ <h2 class="govuk-heading-m">@title</h2>
 }
 
 @content = {
-    <dl class="govuk-summary-list">
-        @govukSummaryList(
-            SummaryList(
-                rows = section.rows
+    @if(section.rows.nonEmpty) {
+        <dl class="govuk-summary-list">
+            @govukSummaryList(
+                SummaryList(
+                    rows = section.rows
+                )
             )
-        )
-    </dl>
+        </dl>
+    }
 
-    @section.children.map(this(_, nestingLevel + 1))
+    @if(section.children.nonEmpty) {
+        @section.children.map(this(_, nestingLevel + 1))
+    }
```

Comment:
Is this change doing anything?

Reply:
Yeah, because we are checking each section independently we are able to add the links to each section without having the formatting of the section rows, without this change we had a large amount of whitespace between the the section header and the links

Reply:
I'll double check :) You're probably right 😁

Reply:
You were right! Removed